packages/renderer/src/lib/dashboard/ProviderConfigured.svelte

```diff
@@ -39,7 +39,11 @@ async function runProvider() {
   runInProgress = false;
 }
 
-onMount(() => {
+onMount(async () => {
+  // Get the autostart configuration setting. If it's disabled, we do *not* want to start the current container provider.
```

Comment:
I think here we can keep what is currently used ( no need to check for autostart property. It's all user driven )